src/random_test.cpp

```diff
@@ -27,6 +27,8 @@ static std::vector<std::string> g_encryption;
 static std::vector<std::string> g_compression = {"none", "zlib", "lz4"};
 static std::vector<std::string> g_row_format;
 static std::vector<std::string> g_tablespace;
+std::vector<std::string> locks;
```

Comment:
Add static here. I forgot. It would be shared resource in multiple thread

Reply:
fixed